director/accounts/templates/accounts/account_profile.html

```diff
@@ -19,35 +19,28 @@
                 </div>
                 <div class="level-right">
                     <p class="level-item">
-                        <a class="button is-success" href="{% url 'project_create' %}?account_id={{ account.pk }}">
+                        <a class="button is-primary is-rounded is-outlined call-to-action" href="{% url 'project_create' %}?account_id={{ account.pk }}">
```

Comment:
I think `call-to-action` is redundant now? What properties do we need for it – I didn't use it throughout any of my last round of changes.